recOrder/acq/acquisition_workers.py

```diff
@@ -61,7 +61,8 @@ def _generate_reconstruction_config_from_gui(
         )
         birefringence_apply_inverse_settings = (
             settings.BirefringenceApplyInverseSettings(
-                wavelength_illumination=calib_window.recon_wavelength / 1000,
+                wavelength_illumination=calib_window.recon_wavelength
+                / 1000,  # um
```

Comment:
This could use more documentation. I'm assuming it means:
```suggestion
                / 1000,  # convert from um to nm
```